.github/workflows/run-notebook-tests-on-last-released-version.yml

```diff
@@ -0,0 +1,40 @@
+name: Notebook-Tests--Against-Last-Release
+
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  run-perf-test:
+    runs-on: 'vs2017-win2016'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    
+    - name: Upgrade pip, setuptools, and wheel before installing other dependencies
+      run: python -m pip install --upgrade pip setuptools wheel
+
+    - run: pip install -r requirements.txt
+      name: 'Install required packages'
+      shell: bash
+    
+    - run: pip install fairlearn
+      name: 'Install last released version of Fairlearn'
```

Comment:
Since I'm paranoid, I'd suggest deleting the fairlearn directory which was just checked out

Reply:
one other thing you can do is to run the notebooks and the tests from `/tmp`

Reply:
Yes. It occurred to me that this could be built on the stage of the release pipeline which tests against the just-deployed pypi package.

Reply:
@riedgar-ms I'm much less worried about the release pipeline because it would reflect the state from master at that point, and that's something we have covered through the other notebook tests.

Deleting the fairlearn directory is certainly not a bad idea.

Reply:
Done